public/js/components/FormFields/FormFieldRadioButtons.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+export default class FormFieldRadioButtons extends React.Component {
+
+  renderButton(value, i) {
+    return (
+      <div key={i} className="form__group form__group--radio">
+        <input className="form__radio-btn" type="radio" name={this.props.fieldName} value={value} defaultChecked={this.props.checkedValue === value ? 'checked' : false} />
```

Comment:
There was a warning that `checked` without an `onClicked` handler would lead to the radio buttons being unable to change.